platforms/node/versionsToBuild.txt

```diff
@@ -39,4 +39,4 @@
 6.17.1
 8.17.0
 10.19.0
-12.16.0
```

Comment:
Do not remove this. We still want to keep the older versions in storage since an end user might explicitly opt-in into it.